test/python/fx_importer/sparsity/sparse_test.py

```diff
@@ -216,25 +216,25 @@ def forward(self, x, v):
     print("torch.mlir   =", res2)
 
 
-@run
+# @run
 #
-# CHECK-LABEL: test_sparse_SpMM
-# CHECK:       #[[$COO:.*]] = #sparse_tensor.encoding<{ map = (d0, d1) -> (d0 : compressed(nonunique), d1 : singleton(soa)), posWidth = 64, crdWidth = 64 }>
-# CHECK:       func.func @main(
-# CHECK-SAME:    %[[A:.*0]]: !torch.vtensor<[8,8],f32,#[[$COO]]>,
-# CHECK-SAME:    %[[B:.*1]]: !torch.vtensor<[8,8],f32>) -> !torch.vtensor<[8,8],f32> {
-# CHECK:         %[[R:.*]] = torch.aten.{{matmul|mm}} %[[A]], %[[B]] : !torch.vtensor<[8,8],f32,#[[$COO]]>, !torch.vtensor<[8,8],f32> -> !torch.vtensor<[8,8],f32>
-# CHECK:         return %[[R]] : !torch.vtensor<[8,8],f32>
-# CHECK:       }
+# C_HECK-LABEL: test_sparse_SpMM
```

Comment:
FYI: FileCheck comment syntax: https://llvm.org/docs/CommandGuide/FileCheck.html#the-com-directive

Reply:
Yeah, I have deliberately kept them as `C_HECK`. I remember once I removed those FileChecks and Aart Bik told me that instead of removing them I can just tweak them like this. Here: https://github.com/llvm/torch-mlir/pull/3340#discussion_r1633543999

What should I do in this case?